libraries/GCS_MAVLink/GCS_FTP.cpp

```diff
@@ -505,7 +483,27 @@ void GCS_MAVLINK::ftp_worker(void) {
                             break;
                         }
 
-                        const uint32_t transfer_size = 100;
+                        /*
+                          calculate a burst delay so that FTP burst
+                          transfer doesn't use more than 1/3 of
+                          available bandwidth on links that don't have
```

Comment:
Does this affect USB connections?

Reply:
no, USB connections are always assumed to have flow control